src/components/auto-complete/auto-complete.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { OptionList } from '../selection-list/selection-list';
+import { SBComponent } from 'stylable-react-component/dist/stylable-react';
+import { root } from 'wix-react-tools';
+import style from './auto-complete.st.css';
+
+export interface AutoCompleteProps extends OptionList {
+    value?: string;
+    onChange?: (value: string) => void;
+    style?: any;
+    className?: string;
+}
+
+@SBComponent(style)
+export class AutoComplete extends React.Component<AutoCompleteProps, {}> {
+
+    public static defaultProps: AutoCompleteProps = { dataSource: [], onChange: () => {}, value: '' };
```

Comment:
@AviVahl fixed
@alisey while this is true, i feel that the user of this component can immediately see its handled this way instead of digging deep will appreciate its presence